tempesta_fw/t/functional/helpers/control.py

```diff
@@ -145,6 +149,17 @@ def parse_out(self, stdout, stderr):
         m = re.search(r'Non-2xx or 3xx responses: (\d+)', stdout)
         if m:
             self.errors = int(m.group(1))
+        m = re.search(r'Requests\/sec:\s+(\d+)', stdout)
+        if m:
+            self.rate = int(m.group(1))
+        m = re.search(r'(Socket errors:.+)\n', stdout)
+        if m:
+            tf_cfg.dbg(0, ("WARNING! Socket errors on wrk. "
```

Comment:
Debug level 0? Shouldn't we print only results of test in debug level 0?

Reply:
Wrk errors can mean a lot: network errors, HTTP parsing errors, and more. Unfortunately we can't get better statistics than `connect`, `read`, `write` or  `timeout` errors. Since normally none of them are acceptable, it seems good point to raise the test failure. Allowing to treat such errors as ordinary one can be also helpful.
I've updated the patch to reflect this behaviour. Any suggestions?